docs/sources/enterprise/aws-secrets-plugin.md

```diff
@@ -0,0 +1,87 @@
+---
+aliases:
+  - /docs/grafana/latest/enterprise/aws-secrets-plugin/
+description: 'Configuring the AWS Secrets Manager plugin'
+keywords:
+  - grafana
+  - secrets
+  - secretsmanager
+  - aws
+  - backend
+title: AWS Secrets Manager Integration
+weight: 2000
+---
+
+# AWS Secrets Manager Plugin
+
+By default, Grafana stores all encrypted secrets in its own SQL database. The AWS Secrets Manager plugin is available for users who wish to store secrets in [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/). In the current version of Grafana, this is limited to datasource secrets, but it will eventually include all secrets (e.g. user credentials).
+
+There are a number of configuration options available that determine secret storage behavior in AWS Secrets Manager. We also provide the ability to migrate secrets from existing Grafana instances to AWS Secrets Manager. We provide a degree of backwards compatibility so that the plugin can be quickly uninstalled, but it has some [limitations](#limitations).
+
+## Installation
+
+You can install the plugin using Grafana CLI:
```

Comment:
```suggestion
You can install the plugin using [Grafana CLI]({{< relref "../../../cli/" >}}):
```

Relative pathing here assumes the document is moved as suggested. If it isn't, the path should be `../cli/`.